nautobot/dcim/lookups.py

```diff
@@ -1,9 +1,33 @@
-from django.contrib.postgres.fields.array import ArrayContains
+from django.db import NotSupportedError
+from django.db.models.lookups import Lookup, Contains
 
 from nautobot.dcim.utils import object_to_path_node
 
 
-class PathContains(ArrayContains):
+class PathContains(Lookup):
```

Comment:
Would it be desirable to add some tests (probably in `nautobot/dcim/tests/test_cablepaths.py` directly exercising the `path__contains=` operator? As far as I can tell from some quick greps this isn't something that currently exists.

Reply:
Yesss feeling good about this now. ;)